quantum/stm32/chconf.h

```diff
@@ -58,7 +58,7 @@
  *          The value one is not valid, timeouts are rounded up to
  *          this value.
  */
-#define CH_CFG_ST_TIMEDELTA                 2
+#define CH_CFG_ST_TIMEDELTA                 0
```

Comment:
For anyone wondering (I was), this is the same change that #6301 makes :)

Reply:
Correct.  This is a change that is needed for PWM to work properly. Not changing it cause the PWM stuff to error out.  

So basically, I only changed it because I had to.

Reply:
Are you sure its a hard requirement? PWMD4 is being used by the leds, and mcuconf is set to `#define STM32_ST_USE_TIMER                  2`. It would error out if you were trying to use PWM2 which at the moment is enabled but to my knowledge not being used. It might be simpler for this PR to just set `#define STM32_PWM_USE_TIM2                  FALSE`

Reply:
Ah, you're correct. 

And mostly, this is an area that I'm really not familiar enough with, and other changes have done the same.